packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -162,6 +162,72 @@ When the `smsRole` property is specified, the `smsRoleExternalId` may also be sp
 assume role policy should be configured to accept this value as the ExternalId. Learn more about [ExternalId
 here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
 
+User pools can be configured to enable MFA. It can either be turned off, set to optional or made required. Setting MFA
```

Comment:
```suggestion
User pools can be configured to enable multi-factor authentication (MFA). It can either be turned off, set to optional or made required. Setting MFA
```